Frameworks/UIKit/WYPopoverController.m

```diff
@@ -70,9 +70,17 @@ - (BOOL)getValueOfRed:(CGFloat *)red green:(CGFloat *)green blue:(CGFloat *)blue
     // model: kCGColorSpaceModelRGB, num_comps: 4
     // model: kCGColorSpaceModelMonochrome, num_comps: 2
     
+    CGColorSpaceModel colorSpaceModel;
+
     CGColorSpaceRef colorSpace = CGColorSpaceRetain(CGColorGetColorSpace([self CGColor]));
-    CGColorSpaceModel colorSpaceModel = CGColorSpaceGetModel(colorSpace);
-    CGColorSpaceRelease(colorSpace);
+
+    if (colorSpace != NULL) {
+        colorSpaceModel = CGColorSpaceGetModel(colorSpace);
+        CGColorSpaceRelease(colorSpace);
+    } else {
+        // Default to RGB in the absence of a colorSpace from CGColor
+        colorSpaceModel = kCGColorSpaceModelRGB;
+    }
```

Comment:
colorSpace NULL is already handled in CGColorSpaceGetModel(), so it should be safe to revert this file?